packages/reporter/src/header/header.tsx

```diff
@@ -41,7 +41,7 @@ const Header = observer(({ appState, events = defaultEvents, statsStore, runnabl
       </button>
     </Tooltip>
     <div className='spacer' />
-    {runnablesStore.testFilter && <DebugDismiss matched={runnablesStore.totalTests} total={runnablesStore.totalUnfilteredTests} />}
+    {runnablesStore.testFilter && runnablesStore.totalTests > 0 && <DebugDismiss matched={runnablesStore.totalTests} total={runnablesStore.totalUnfilteredTests} />}
```

Comment:
No test to validate this?

Reply:
I am not sure of a way to test this ephemeral state. I am not actually sure how the runnables-store gets in this state.  It looks like in the `setRunnables` method that the `this._tests` map gets set in the first line of the method in the call to `_createRunnableChildren`.  Then the `totalTests` gets set a few lines later that counts the length of this, and finally the `testFilter` is set almost at the end. Any ideas on how to test that?

Reply:
I agree this isn't worth it. I get the value of testing everything, but the cost of either

1) mocking the universe to do so
2) making a big refactor to do so

I especially have come to dislike (1) - often this leads to tests that pass, even if the actual feature does regress. The cost of writing the test seems to out weight the actual benefit of the test. If the bug was more breaking, I'd push for this - but I don't know how likely this really is to regress, and if it *does*, it really wouldn't be the end of the world.